README.md

```diff
@@ -644,20 +645,34 @@ callback function as a parameter as well as `offRequest` and `offResponse`
 functions which will remove a callback function from the `onRequest` or 
 `onResponse` set if it exists. 
 
-The `onRequest(callback)` function takes a `callback` function that will pass the xhr `request`
-Object to that callback. These callbacks are called synchronously, in the order registered
-and act as pre-request hooks for modifying the xhr `request` Object prior to making a request.
+The `onRequest(callback)` function takes a `callback` function that will pass the xhr `options`
+Object to that callback. These callbacks are called synchronously, in the order registered 
+and act as pre-request hooks for modifying the xhr `options` Object prior to making a request.
 
 Example:
 ```javascript
-const playerRequestHook = (request) => {
-  const requestUrl = new URL(request.uri);
+const playerRequestHook = (options) => {
```

Comment:
One thought on these samples, should they be wrapped in an `xhr-hooks-ready` event callback to show the recommended approach?

```js
player.on('xhr-hooks-ready', () => {
  player.tech().vhs.xhr.onRequest((options) => options);
});
```

Reply:
Good call, will add examples with the event.

Reply:
Added the `'xhr-hooks-ready'` event listener to all the per-player examples.